StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/ElementsCustomer.swift

```diff
@@ -36,6 +36,8 @@ struct ElementsCustomer: Equatable, Hashable {
         }
 
         // Optional
+        // to test default payment methods reading from back end, hard-code a valid default payment method
+        // later, when API calls to get and update default payment method are available, that will no longer be needed
```

Comment:
This comment doesn't make much sense to me, I don't see any code changes near this comment indicating we are hard coding a default PM.

Reply:
What I was trying to say is that the only way I knew to test it was to add a payment method, get its stripe ID, and then hardcode that here so that the elements session will set that as its default. And then after adding other cards, even after paying with them, when the payment sheet is reloaded, it will have the hardcoded default selected, which shows that it's reading from the elements session instead of the local default.

Right now, there's a duping process where when you read from "default_payment_method", it returns the locally saved default that you pass into the API call— it's not actually retrieving a default from the back end. When the API endpoints are ready, then we should be able to actually get the value stored in the back end, but until then, the only way I could think to test that it's reading from the elementsSession value instead of the local default was to hard-code a default (e.g. let defaultPaymentMethod = "pm_1QSOUILu5o3P18ZpN4PY49SE")